src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -716,21 +718,24 @@ def __call__(
                 # broadcast to batch dimension in a way that's compatible with ONNX/Core ML
                 timestep = t.expand(latents.shape[0]).to(latents.dtype)
```

Comment:
This would be another candidate for syncs because we're iterating over a tensor here. See this internal [link](https://huggingface.slack.com/archives/C068ZAHJZCZ/p1724793242004719?thread_ts=1723714976.523789&cid=C068ZAHJZCZ) for more details.